### PR TITLE
Enable travis

### PR DESCRIPTION
**Pull request by languitar**
This PR integrates the project with travis for cpp, python and java. CL is still missing.

### CODE REVIEW

File: test/integrationtest.py [hunk 1]
```diff
@@ -103,7 +103,7 @@ def __init__(self, command):
         time.sleep(2)
 
     def __del__(self):
-        print("Stopping command %s" % self.__open)
+        print(("Stopping command %s" % self.__open))
```
Comment (scymtym) [inline test/integrationtest.py]: What does this do?

File: test/integrationtest.py [hunk 2]
```diff
@@ -487,28 +489,25 @@ def run():
 
     # Add a test method for the configuration test for each language.
     if "parser" in selectedTests:
-        map(IntegrationTest.addParserTest, selectedLanguages)
+        list(map(IntegrationTest.addParserTest, selectedLanguages))
```
Comment (scymtym) [inline test/integrationtest.py:501]: When is `map` permitted? Previous uses where changed to `[… for … in …]`.

**Review by scymtym — CHANGES_REQUESTED** [review]
Basically LGTM. Minor issues in code comments.

File: .travis.yml [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+  - wget "https://dl.bintray.com/open-rsx/travis-artifacts/rsb-python-master.tar.gz"
+  - "tar -xzf rsb-python-*.tar.gz"
+  - cd ..
+install:
+  - ls upstream
+  - pip install wheel xmlrunner
+  - pip install upstream/rsb_python-*.whl
+script:
+  - mkdir -p build && cd build
+  - pwd
```
Comment (scymtym) [inline .travis.yml]: Maybe remove this?

File: test/integrationtest.py (hunk 1 above)
Comment (languitar) [reply test/integrationtest.py]: I didn't think I changed something here, but it's just wrong and will print out the string inside a tuple with length one ;-)

File: test/integrationtest.py (hunk 2 above)
Comment (languitar) [reply test/integrationtest.py:501]: people usually prefer list comprehension, but map or filter etc are not strictly forbidden

**Comment by languitar** [comment]
I have addressed the important issues, map and list will be gone with the next commit in any case.

**Review by scymtym — APPROVED** [review]

**Pull request merged**